coverage/.resultset.json

```diff
@@ -47,6 +47,6 @@
         null
       ]
     },
-    "timestamp": 1574094123
+    "timestamp": 1574094491
```

Comment:
This will be adding back in the coverage folder. Can you take this out before doing another pull request.